flair/models/text_classification_model.py

```diff
@@ -89,26 +78,20 @@ def load_from_file(cls, model_file):
         # ATTENTION: suppressing torch serialization warnings. This needs to be taken out once we sort out recursive
         # serialization of torch objects
         warnings.filterwarnings("ignore")
-        state = torch.load(model_file, map_location={'cuda:0': 'cpu'})
```

Comment:
I believe this is needed if a model was trained on GPU but is loaded on CPU

Reply:
It is not working if you load the GPU model on a GPU machine again. I'll try to load a model trained on GPU on a CPU machine.